File: 125-valid-palindrome/valid-palindrome.cpp
class Solution {
public:
    bool isPalindrome(string s) {
        string t = "";
        for(char ch : s){
            if(isalpha(ch)){
                if(ch >= 'A' && ch <= 'Z')
                    ch = tolower(ch);
                t += ch;
            }else if(ch>='0' && ch<='9'){
                t += ch;
            }
        }

        
        // cout<<t<<endl;
        int i = 0, j = t.size()-1;
        while(i<j){
            if(t[i] == t[j]){
                i++;
                j--;
            }else{
                return false;
            }
        }
        return true;
    }
};

File: 125-valid-palindrome/valid-palindrome.py
class Solution(object):
    def isPalindrome(self, s):
        """
        :type s: str
        :rtype: bool
        """
        st = ""
        for ch in s:
            if ch.isalnum():
                st += ch.lower()

        j = len(st)-1
        i = 0
        while(i <= j):
            if st[j] == st[i]:
                j -= 1
                i += 1
            else:
                return False
        return True

        